src/components/Questions.vue

```diff
@@ -15,7 +15,55 @@
                 </tr>
             </tbody>
             </template>
-        </v-simple-table>
```

Comment:
Go ahead and remove the refactored code, no need to comment it out.  We have it in github commit history if we ever need to reference or roll back.

Reply:
Removed </v-simple-table>